samples/bootloader/README.rst

```diff
@@ -33,8 +33,10 @@ This is accomplished by the following steps:
      This metadata contains the full public key corresponding to the private key that was used to sign the firmware.
      The bootloader sample checks the public key against a set of provisioned keys.
      Note that to save space, only the hashes of the provisioned keys are stored, and only the hashes of the keys are compared.
-     If the public key in the metadata matches one of the provisioned public key hashes, the image is considered valid.
-     If the public key does not match any of the provisioned hashes, validation fails.
+     If the public key in the metadata matches one of the provisioned public key hashes that are still valid, the image is considered valid.
```

Comment:
`... matches one of the ... that are ` -> `that is` ? 

 I'm not good at this, but could be something.

Reply:
"are" refers to public key hashes (plural) so I think "are" is correct.